kindle_notes_to_md.py

```diff
@@ -100,9 +98,10 @@ def parse_file(self, html_file: str):
       # Strange roles are for strange people!
       # </div>
       elif c == 'noteHeading':
+        # first figure out what location this note/highlight is for
+        source = ' '.join(div.stripped_strings)
```

Comment:
@thomaswinged is there a reason to move this out of the `try` block?

Reply:
Yes, `source` variable would not be invalid in line 138 in case there is an error in try catch block. I moved it out of try catch block to make sure it's initialized.